packages/webview_flutter/webview_flutter_android/android/build.gradle

```diff
@@ -52,6 +52,7 @@ android {
         implementation 'androidx.annotation:annotation:1.9.1'
         implementation 'androidx.webkit:webkit:1.12.1'
         testImplementation 'junit:junit:4.13.2'
+        testImplementation 'org.mockito:mockito-core:5.16.1'
```

Comment:
This was required for me to run tests locally. I'm not sure if this is just me or not.